vespa/application.py

```diff
@@ -144,6 +148,55 @@ def feed_data_point(self, schema: str, data_id: str, fields: Dict) -> Response:
         response = http.post(end_point, json=vespa_format, cert=self.cert)
         return response
 
+    async def async_feed_data_point(self, session: aiohttp.ClientSession, schema: str, data_id: str, fields: Dict):
+        """
+        Feed a data point to a Vespa app using asyncio/aiohttp.
+
+        :param session: The aiohttp client session
+        :param schema: The schema that we are sending data to.
+        :param data_id: Unique id associated with this data point.
+        :param fields: Dict containing all the fields required by the `schema`.
+        :return: Response of the HTTP POST request.
+        """
+        end_point = "{}/document/v1/{}/{}/docid/{}".format(
+            self.end_point, schema, schema, str(data_id)
+        )
+        vespa_format = {"fields": fields}
+        sslcontext = False
+        if self.cert is not None:
+            sslcontext = ssl.create_default_context(cafile=self.cert)
```

Comment:
We should add a cloud integration test to check if the `sslcontext` works as expected.

Reply:
Fixed.